src/pages/CreateNFTCollection.vue

```diff
@@ -14,19 +14,28 @@ import LSP12IssuedAssetsSchema from '@erc725/erc725.js/schemas/LSP12IssuedAssets
 import LSP8Mintable_0_5_0 from '../contracts/LSP8Mintable_0_5_0.json';
 
 import { IPFS_GATEWAY_API_BASE_URL, IPFS_GATEWAY_BASE_URL, BLOCKCHAIN_EXPLORER_BASE_URL } from '../constants';
+import { addLuksoL14Testnet, addLuksoL16Testnet } from '../../swap-network';
 
 const deploying = ref(false);
 const error = ref(false);
 const isEOA = ref(false);
 const deployEvents = ref([]);
 const isSuccess = ref(false);
+const wrongNetwork = ref(false);
 
 // Form fields
 const tokenName = ref('');
 const tokenSymbol = ref('');
 const description = ref('');
 
 async function onSubmit(e) {
+  // Check network
+  let chainID = await web3.eth.getChainId();
+  if (chainID !== 22 && chainID !== 2828) {
```

Comment:
perfect example of why it makes sense to use the reco from the comment above.

Put these variables (22, 2828) in a global conf file.

1) It gives them a name
2) You don't duplicate this value in various files